playwright/src/test/java/com/microsoft/playwright/TestTap.java

```diff
@@ -97,6 +98,7 @@ void shouldNotSendMouseEventsWhenTouchendIsCanceled() {
   }
 
   @Test
+  @Disabled("noWaitAfter was removed from tap() in https://github.com/microsoft/playwright/pull/31739")
```

Comment:
In .NET I don't pass the values to the server anymore except for press/click/selectOption and adjusted the tests like we did upstream instead of disabling them.